testing/ostest/mutex.c

```diff
@@ -103,11 +116,172 @@ static void *thread_func(FAR void *parameter)
   return NULL; /* Non-reachable -- needed for some compilers */
 }
 
+static void *moved_mutex_thread_func(FAR void *parameter)
+{
+  FAR mutex_thread_args_t *args = (FAR mutex_thread_args_t *)parameter;
+
+  int id = args->thread_id;
+  int ndx = id - 1;
+  int i;
+  unsigned long local_loops = 0;
```

Comment:
```suggestion
  i local_loops;
```

Reply:
Something doesn't seem right with this suggestion.

Did you mean:

```suggestion
  int local_loops;
```

?

Reply:
By the way, if changing `local_loops` to `int` here, then `nloops` of `mutex_thread_args_t` needs to be `int` also? Or is it better to leave as unsigned long?

Reply:
All types changed to "int", I guess it's OK for such a simple test

Reply:
> Something doesn't seem right with this suggestion.
> 
> Did you mean:
> 
> ?

Yes, the reason is that NLOOPS is type of int const.